lib/services/registered_addons.pm

```diff
@@ -69,7 +69,13 @@ sub check_suseconnect {
             my $iden = $_->{identifier};
             my $status = $_->{status};
             push(@addons, $iden);
-            die "$iden register status is: $status" if ($status ne 'Registered');
+            if ($status ne 'Registered') {
+                if ($iden eq 'sle-module-packagehub-subpackages') {
```

Comment:
Could you move the l73 and 74 to 71? We don't need to judge the not registered state for sle-module-packagehub-subpackages to workaround,  And here just easily judge not registered then die.